critiquebrainz/data/model/user.py

```diff
@@ -111,9 +112,12 @@ def stats(self):
             votes_this_month=self.votes_since_count(date(today.year, today.month, 1)))
 
     def _reviews_since(self, date):
-        rev_q = db.session.query(Revision.review_id, db.func.min(Revision.timestamp).label('creation_time'))\
-            .group_by(Revision.review_id).subquery('time')
-        return self._reviews.outerjoin(rev_q, Review.id == rev_q.c.review_id).filter(rev_q.c.creation_time >= date)
+        rev_q = db.session.query(Revision.review_id,
+                                 db.func.min(Revision.timestamp).
+                                 label('creation_time')). \
```

Comment:
Shouldn't the trailing `.` go at the beginning of the next line? That's how you've done elsewhere AFAICT.